enhancements/declarative-index-config.md

```diff
@@ -0,0 +1,947 @@
+
+```bash
+$ cd community-operators
+$ tree
+.
+├── etcd.json
+├── amqstreams.json
+
+$ cat etcd.json
+{
+    "schema": "olm.package",
+    "name": "etcd",
+    "defaultChannel": "singlenamespace-alpha",
+    "icon": {
+        "base64data":"iVBORw0KGgoAAAANSUhEUgAAA.....",
+        "mediatype":"image/png"
+    },
+    "channels": ["alpha", "singlenamespace-alpha", "clusterwide-alpha"],
+    "description": "A message about etcd operator, a description of channels"
+},
+{
+    "schema": "olm.bundle", 
+    "name": "etcdoperator-community.v0.6.1",
+    "package": "etcd",
+    "image": "quay.io/operatorhubio/etcd:v0.6.1",
+    "version": "0.6.1",
+    "properties":[
+        {
+            "name": "olm.package",
+            "values": {
+                "packageName": "etcd",
+                "version": "0.6.1"
+            }
+        },
+        {
+            "name":"olm.gvk",
+            "type": "provided",
+            "values":{
+                "group": "etcd.database.coreos.com", 
+                "kind": "EtcdCluster", 
+                "version": "v1beta2"
+            }
+        },
+        {
+            "name": "olm.channel",
+            "value": "alpha"
+        }
+    ],
+    "relatedImages": [
+        {
+            "name": "etcdv0.6.1",
+            "image": "quay.io/coreos/etcd-operator@sha256:bd944a211eaf8f31da5e6d69e8541e7cada8f16a9f7a5a570b22478997819943"
+        }
+    ]
+},
+{
+    "schema": "olm.bundle",
+    "name": "etcdoperator.v0.9.0",
+    "package": "etcd",
+    "image": "quay.io/operatorhubio/etcd:v0.9.0",
+    "version": "0.9.0",
+    "properties":[
+        {
+            "name": "olm.package",
+            "values": {
+                "packageName": "etcd",
+                "version": "0.9.0"
+            }
+        },
+        {
+            "name":"olm.gvk",
+            "type": "provided",
+            "values":{
+                "group": "etcd.database.coreos.com", 
+                "kind": "EtcdBackup", 
+                "version": "v1beta2"
+            }
+        },
+        {
+            "name": "olm.channel",
+            "value": "singlenamespace-alpha"
+        }
+        {
+            "name": "olm.channel",
+            "value": "clusterwide-alpha"
+        }
+    ],
+    "relatedImages" : [
+        {
+            "name": "etcdv0.9.0",
+            "image": "quay.io/coreos/etcd-operator@sha256:db563baa8194fcfe39d1df744ed70024b0f1f9e9b55b5923c2f3a413c44dc6b8"
+        }
+    ]
+},
+{
+    "schema": "olm.bundle",
+    "name": "etcdoperator.v0.9.2",
+    "package": "etcd",
+    "image": "quay.io/operatorhubio/etcd:v0.9.2",
+    "version": "0.9.2",
+    "properties":[
+        {
+            "name": "olm.package",
+            "values": {
+                "packageName": "etcd",
+                "version": "0.9.2"
+            }
+        },
```

Comment:
This seems like duplicative information, since we already capture package name and version in the root of the `olm.bundle` schema. Is it necessary for this property to be explicitly included?

Also related. I didn't see any mention of `dependencies.yaml` in this EP. Do package dependencies need to be accounted for? If so, is there an example for how that looks?